RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -7082,13 +7203,3 @@ int poolSkillPracticeGains()
 	return count;
 }
 
-void resetThisLoop()
-{
-	//These settings should never persist into another turn, ever. They only track something for a single instance of the main loop.
-	//We use boolean instead of adventure count because of free combats.
-	
-	set_property("auto_doCombatCopy", "no");
-	set_property("_auto_thisLoopHandleFamiliar", false);	//have we called handleFamiliar this loop
-	set_property("auto_disableFamiliarChanging", false);	//disable autoscend making changes to familiar
-	set_property("auto_familiarChoice", "");				//which familiar do we want to switch to during pre_adventure
-}
```

Comment:
why is this being removed?

Reply:
also if it is to be removed it needs to be removed from header file and autoscend as well.

Reply:
This was accidentally not undone when I copied these changes over from #453. Will fix.